lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -174,8 +174,12 @@ def move_lower
           return unless lower_item
 
           acts_as_list_class.transaction do
-            lower_item.decrement_position
-            increment_position
+            if lower_item.send(position_column) != self.send(position_column)
```

Comment:
Yes, it is possible, but didn't want to change default behaviour, just handle "non-consequence" branch, so consider that as a refactor :).